night-light-slider.timur@linux.com/extension.js

```diff
@@ -98,7 +98,7 @@ const SliderMenuItem = new Lang.Class({
 
 // Extension initilization
 function Extension () {
-  this.enable = function enable () {
+  this.enable = () => {
```

Comment:
Doing this so I have `this` scope for settings `this.indicators`, which I use to check if I should restore the night light icon on extension `disable` function. This is hacky, but temporary feature in place while I refactor the whole extension.